### DISCUSSION TITLE
Meeting 2024-11-12

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#804] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-11-12)

**Comment by berkus** [comment]
- Add @berkus to maintainers in stm32-rs org. Per @burrbull previous requests.

I have some stm32 boards at home and can test and review contribs.

**Comment by MabezDev** [comment]
Possibly a spicy topic: I'd like to discuss HAL maintainers agreeing to ditch embedded-hal@0.2 impls in their HALs to promote drivers being updated to embedded-hal@1.0.0.

**Comment by BartMassey** [comment]
Save the date: next Rust Embedded Discovery Book Sprint scheduled for January 25.
